hazelcast/src/main/java/com/hazelcast/internal/metrics/impl/CompositeMetricsCollector.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.metrics.impl;
+
+import com.hazelcast.internal.metrics.MetricDescriptor;
+import com.hazelcast.internal.metrics.collectors.MetricsCollector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class CompositeMetricsCollector implements MetricsCollector {
+
+    private final List<MetricsCollector> collectors;
+
+    public CompositeMetricsCollector(MetricsCollector... collectors) {
+        this.collectors = new ArrayList<>(asList(collectors));
```

Comment:
Does additional list creation (`new ArrayList<>()`) provide any benefits when compared with the following?
```java
this.collectors = asList(collectors);
```

Reply:
No, it was a leftover/mistake. Changed to the simple `asList()`.